source/includes/_172-merged-environment-config.md.erb

```diff
@@ -0,0 +1,6 @@
+# Merged Environment Config
+
+The merged environment config API allows users with administrator role to manage config repo environment config.
```

Comment:
`manage` is a misleading word. All you are doing is displaying info.And, `config repo environment config` is a little vague to me.